docs/technology-choices.md

```diff
@@ -0,0 +1,166 @@
+![zero](https://raw.githubusercontent.com/commitdev/zero/main/docs/img/logo-small.png)
+
+## Technology Choices
+As we add features to Zero, we rely heavily on our years of experience with founding and growing startups, and judge tools and technologies based on the axes of:
+- Quality - Is it the best tool for the job? Will it allow a project to start small and scale big?
+- Simplicity - Is it easy to set up automatically and in a way that is easy to understand and maintain for the developers and operators of the project?
+- Price - Ideally we look for open source tools, but there are some tools we integrate with that have a cost if we see it as providing enough value to justify that cost. In this case we will often try to also include an open source alternative.
+
+When there are multiple technologies that we consider to be front-runners, we try to add multiple options, along with documentation and use cases to describe in which situations each tool might be the right choice for a project.
+
+### Core principles
+**[Twelve-Factor App](https://12factor.net/)**
+
+The Twelve-Factor App is a methodology for building Software-as-a-Service that encourages building applications in a way that inherently makes them more stable, secure, failure-safe, easier to measure, and easier to reason about. As much as possible we try to stick with these principles when building Zero, as we have seen it to be a very effective way to build scalable, manageable applications.
+
+**[CNCF](https://www.cncf.io/)**
```

Comment:
I am not sure if CNCF should be under the "Core principles" category. I would say that the Zero team supports open source and writing and consuming OSS software is the principle here.